lib/puppet/configurer.rb

```diff
@@ -161,6 +161,7 @@ def run_internal(options)
         begin
           if node = Puppet::Node.indirection.find(Puppet[:node_name_value],
               :environment => Puppet::Node::Environment.remote(@environment),
+              :configured_environment => Puppet[:environment],
```

Comment:
They're the same here, later calls they aren't. I added it because I was testing with different logic in the node terminus, and it made sense to me to be explicit, but in this case we could drop it and reduce network activity slightly.